### ISSUE TITLE
color_only_init error

### ISSUE THREAD

**Issue by Phlya** [deeptools/pyGenomeTracks#31]
When using pygenometracks to plot Hi-C data (at least from cooler files) I get an error:
```
INFO:root:Generating grammar tables from /usr/lib/python3.5/lib2to3/Grammar.txt
INFO:root:Generating grammar tables from /usr/lib/python3.5/lib2to3/PatternGrammar.txt

title not set for 'section 3. [Genes]'
Traceback (most recent call last):
  File "/usr/local/bin/pyGenomeTracks", line 11, in <module>
    main(args)
  File "/usr/local/lib/python3.5/dist-packages/pygenometracks/plotTracks.py", line 279, in main
    track_label_width=args.trackLabelFraction)
  File "/usr/local/lib/python3.5/dist-packages/pygenometracks/tracksClass.py", line 102, in __init__
    self.track_obj_list.append(track_class(properties))
  File "/usr/local/lib/python3.5/dist-packages/pygenometracks/tracks/HiCMatrixTrack.py", line 50, in __init__
    self.hic_ma = HiCMatrix.hiCMatrix(self.properties['file'], color_only_init=True)
TypeError: __init__() got an unexpected keyword argument 'color_only_init'

```

**Comment by fidelram** [comment]
It should be `cooler_init_only` 
@joachimwolff can you take a look at this? Do you think we can add a test for cooler?

**Comment by joachimwolff** [comment]
Again this error? I thought it is fixed with 2.1 version. I will have a look at it.

**Comment by bgruening** [comment]
@Phlya  please give the latest version a try. It should hopefully solve your issue.
You can try the newest version here: https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/pygenometracks/pygenomeTracks/
